xdsl/transforms/printf_to_llvm.py

```diff
@@ -103,7 +103,7 @@ def _construct_global(self, val: str):
             _key_from_str(val),
             constant=True,
             linkage="internal",
-            value=builtin.DenseIntOrFPElementsAttr.from_list(t_type, data),
+            value=builtin.DenseIntOrFPElementsAttr([t_type, builtin.BytesAttr(data)]),
```

Comment:
@AntonLydike could you check that this is reasonable or suggest a fix if it is not

Reply:
If tests pass then it's good enough for me!